toolkit/tools/internal/randomization/randomization.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package randomization
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+// RandomString generates a random string of the length specified
```

Comment:
>length [](start = 49, length = 6)

It might be worth adding a word or two clarifying that this is the length in runes, not in bytes, so `len(output)` might not match `lenght`.